wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+    }
+
+    # Motif Analysis
+    if (run_motifs) {
+        call motifs.motifs { input:
+            reference = reference,
+            reference_index = samtools_faidx.faidx_file,
+            bedfile = macs.peakbedfile,
+            motif_databases = motif_databases,
+            default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/MOTIFS",
+        }
+
+        call util.flankbed { input:
+            bedfile = macs.summitsfile,
+            default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/MOTIFS",
+        }
+
+        call motifs.motifs as flank { input:
+            reference = reference,
+            reference_index = samtools_faidx.faidx_file,
+            bedfile = flankbed.flankbedfile,
+            motif_databases = motif_databases,
+            default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/MOTIFS",
+        }
+    }
+
+    call viz.visualization { input:
+        wigfile = macs.wigfile,
+        chromsizes = samtools_faidx.chromsizes,
+        xlsfile = macs.peakxlsfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/NARROW_peaks" + "/" + sub(basename(macs.peakbedfile), "_peaks.bed", ""),
+    }
+
+    call viz.visualization as vizall { input:
+        wigfile = all.wigfile,
+        chromsizes = samtools_faidx.chromsizes,
+        xlsfile = all.peakxlsfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/NARROW_peaks" + "/" + sub(basename(all.peakbedfile), "_peaks.bed", ""),
+    }
+
+    call viz.visualization as viznomodel { input:
+        wigfile = nomodel.wigfile,
+        chromsizes = samtools_faidx.chromsizes,
+        xlsfile = nomodel.peakxlsfile,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/NARROW_peaks" + "/" + sub(basename(nomodel.peakbedfile), "_peaks.bed", ""),
+    }
+
+    call viz.visualization as vizsicer { input:
+        wigfile = sicer.wigfile,
+        chromsizes = samtools_faidx.chromsizes,
+        default_location = sub(basename(sample_bam), ".sorted.b.*$", "") + "/COVERAGE_files/BROAD_peaks",
+    }
+
+    call bedtools.bamtobed as finalbed { input: bamfile = sample_bam }
+
+    call sortbed.sortbed { input: bedfile = finalbed.bedfile }
+
+    call bedtools.intersect { input:
+        fileA = macs.peakbedfile,
+        fileB = sortbed.sortbed_out,
+        countoverlap = true,
+        sorted = true,
+    }
+
+    ### ---------------------------------------- ###
+    ### ------------ S E C T I O N 4 ----------- ###
+    ### ---------- Summary Statistics ---------- ###
+    ### ---------------------------------------- ###
+
+    String string_qual = ""  #buffer to allow for optionality in if statement
+
+    #SUMMARY STATISTICS
+    if (one_fastq) {
+        call util.evalstats as uno_summarystats {
+            # SUMMARY STATISTICS of sample file (only 1 sample file provided)
+            input:
```

Comment:
Ah, this is the interrupting comment instance. I still think `input:` should get moved to precede the comment and directly follow the `{`.

Reply:
Ultimately, I think comment should precede the `call`, but without analyzing the comment content, there's no way to know that in the formatter.

Reply:
Mentioned in a prior comment, but I think you might be onto something and there might be "comment shuffling" we can do, but it feels very OOS for this first PR because it would be quite complicated.